skip_title_check.txt

```diff
@@ -0,0 +1,2 @@
+# Put files you want to skip "titles:" check here:
```

Comment:
"title:" check; we don't want to confuse people as to what the attribute is actually called.